packages/nuxt/src/app/components/nuxt-link.ts

```diff
@@ -207,43 +280,11 @@ export function defineNuxtLink (options: NuxtLinkOptions) {
         required: false,
       },
     },
+    useLink: useNuxtLink,
```

Comment:
What does this do?

Reply:
This matches `RouterLink` behaviour (which exposes a 'bound' composable which other libraries, like vuetify, can use to create custom links).